### PR TITLE
Update server-level-roles.md

### PR DESCRIPTION
**Pull request by MikeyBronowski**
This update relates to #4198. The note has been made in this article [bulk-insert-transact-sql](https://docs.microsoft.com/en-us/sql/t-sql/statements/bulk-insert-transact-sql), but I think [server-level-roles](https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/server-level-roles) should have the information as well.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@MikeyBronowski : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
#label:"aq-pr-triaged"

File: docs/relational-databases/security/authentication-access/server-level-roles.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ monikerRange: ">=aps-pdw-2016||>=sql-server-2016||>=sql-server-linux-2017||=azur
...
 |**processadmin**|Members of the **processadmin** fixed server role can end processes that are running in an instance of [!INCLUDE[ssNoVersion](../../../includes/ssnoversion-md.md)].|  
 |**setupadmin**|Members of the **setupadmin** fixed server role can add and remove linked servers by using [!INCLUDE[tsql](../../../includes/tsql-md.md)] statements. (**sysadmin** membership is needed when using [!INCLUDE[ssManStudio](../../../includes/ssmanstudio-md.md)].)|  
-|**bulkadmin**|Members of the **bulkadmin** fixed server role can run the `BULK INSERT` statement.|  
+|**bulkadmin**|Members of the **bulkadmin** fixed server role can run the `BULK INSERT` statement.<br /><br /> **Note:** **bulkadmin** role or ADMINISTER BULK OPERATIONS permissions is not supported for SQL Server on Linux. Only the sysadmin can perform bulk inserts for SQL Server on Linux. |  |
```
Comment (VanMSFT) [inline docs/relational-databases/security/authentication-access/server-level-roles.md:51]:
```suggestion
|**bulkadmin**|Members of the **bulkadmin** fixed server role can run the `BULK INSERT` statement.<br /><br /> The **bulkadmin** role or ADMINISTER BULK OPERATIONS permissions is not supported for SQL Server on Linux. Only the **sysadmin** can perform bulk inserts for SQL Server on Linux. |  |  
```

**Comment by PRMerger5** [comment]
@VanMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by VanMSFT** [comment]
Hi @MikeyBronowski - Thanks for making the additions! I will approve this.

#sign-off

**Pull request merged**